### ISSUE TITLE
"State Management" section has almost no description

### ISSUE THREAD

**Issue by mushketyk** [onyx-platform/onyx-starter#9]
It's hard to follow without prior Onyx knowledge.

**Comment by MichaelDrogalis** [comment]
@mushketyk I don't see anywhere in this repo where state management is mentioned. Can you directly link me to the spot?

**Comment by mushketyk** [comment]
@MichaelDrogalis I was referring to this section: https://github.com/onyx-platform/onyx-starter/blob/0.8.x/WALKTHROUGH.md#state-management

**Comment by MichaelDrogalis** [comment]
Ah! See, I told you I forget about that document easily ;) Thanks, Ill add more context tonight.
